### ISSUE TITLE
Paginate until no records or limit

### ISSUE THREAD

**Issue by ferrumboll** [Widen/tap-rest-api-msdk#41]
Hello,
There are some APIs that don't have a next page identifier in the response, the only way is to keep querying until a manual limit or until the API gives 0 items.

Is this possible with this Tap?

**Comment by ferrumboll** [comment]
I could try to do a PR with an extra Paginator for this, if it is not possible right now.

**Comment by jlloyd-widen** [comment]
I do not believe that is supported, so if you are able to add a PR, that would be the quickest path to adding support for it.

**Comment by edgarrmondragon** [comment]
fwiw starting with [0.35.0](https://github.com/meltano/sdk/releases) (see https://github.com/meltano/sdk/pull/1918) the sdk will exit the pagination loop if the page contains no records :)
